boltdesignsystem/bolt
# Icon Refactor: Docs Updates

## Jira
N/A

## Summary
Docs-site and demo-specific updates related to the Icon component work in #1602.

## Details
Note: without the other component-specific and build-process specific updates, these docs updates won't work (or pass the build) since current schema validation results prevent custom SVG icons that are not part of the Icon component's schema to get used (the upstream work changes that).

## How to test
Check out the [updated Docs here](https://feature-auto-add-icons.boltdesignsystem.com/pattern-lab/?p=viewall-components-icon) since this PR's deployed code will be broken until the schema changes are merged in.
